src/UserService.Models.Dto/Requests/User/EditUserRequest.cs

```diff
@@ -14,11 +14,11 @@ public class EditUserRequest
         public string MiddleName { get; set; }
         public string About { get; set; }
         public UserGender Gender { get; set; }
-        public string DateOfBirth { get; set; }
+        public DateTime DateOfBirth { get; set; }
```

Comment:
hey should be 'DateTime?' because it is valueType and front Team want to send null that delete it

Reply:
StartWorkingAt too

Reply:
here I think just need to add delete op in validator